anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/detectormapper/DetectorMapping.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2018-2019 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.adaptivealerting.anomdetect.detectormapper;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+/**
+ * The type Detector mapping.
+ * <p>
+ * searchIndexes: index of matching metric-tag in request batch of metric-tags
+ */
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class DetectorMapping {
+    private String id;
+    private Detector detector;
```

Comment:
The ES response has following structure. 
```json {
    "detector": {
      "uuid": "2c49ba26-1a7d-43f4-b70c-c6644a2c1689"
    },
    "expression": {
      "operator": "AND",
      "operands": [
   ```
IMP keeping detector abstraction will make it easier to modify this when we want to utilise other detector related information like type in ad-mapper. to different topics for internal and external detectors.

Otherwise we should change this in ES first and then i can raise a follow-up PR which does this. It will also impact all the tests as well.